distribution/scripts/cloudformation/templates/ec2.yaml

```diff
@@ -17,414 +17,429 @@
-    ConstraintDescription: Must be the URL of the github repository.
-    Default: "https://github.com/ldclakmal/ballerina-performance-aws-ecs.git"
-  GitHubRepoBranch:
-    Type: String
-    Description: Branch of the github repo of the ballerina performance tests.
-    Default: "master"
-  BallerinaZipURL:
-    Type: String
-    Description: URL for the Ballerina Zip file.
-    ConstraintDescription: Must be the URL of the ballerina zip file.
-    Default: "https://dist.ballerina.io/downloads/swan-lake-preview2/ballerina-swan-lake-preview2.zip"
-  JMeterOptions:
-    Type: String
-    Description: Options for JMeter.
-    ConstraintDescription: A string of options for JMeter.
-    Default: "-u 100 -b 512 1024 2048 -d 600 -w 150 -i h1c"
-  BallerinaMemory:
-    Type: String
-    Description: Memory Constraint for the test.
-    Default: "2048"
-  BallerinaCPU:
-    Type: String
-    Description: CPU Constraint for the test.
-    Default: "1024"
-  InstanceType:
-    Type: String
-    Description: WebServer EC2 instance type.
-    Default: t2.micro
-    AllowedValues:
-      - t1.micro
-      - t2.nano
-      - t2.micro
-      - t2.small
-      - t2.medium
-      - t2.large
-      - m1.small
-      - m1.medium
-      - m1.large
-      - m1.xlarge
-      - m2.xlarge
-      - m2.2xlarge
-      - m2.4xlarge
-      - m3.medium
-      - m3.large
-      - m3.xlarge
-      - m3.2xlarge
-      - m4.large
-      - m4.xlarge
-      - m4.2xlarge
-      - m4.4xlarge
-      - m4.10xlarge
-      - c1.medium
-      - c1.xlarge
-      - c3.large
-      - c3.xlarge
-      - c3.2xlarge
-      - c3.4xlarge
-      - c3.8xlarge
-      - c4.large
-      - c4.xlarge
-      - c4.2xlarge
-      - c4.4xlarge
-      - c4.8xlarge
-      - g2.2xlarge
-      - g2.8xlarge
-      - r3.large
-      - r3.xlarge
-      - r3.2xlarge
-      - r3.4xlarge
-      - r3.8xlarge
-      - i2.xlarge
-      - i2.2xlarge
-      - i2.4xlarge
-      - i2.8xlarge
-      - d2.xlarge
-      - d2.2xlarge
-      - d2.4xlarge
-      - d2.8xlarge
-      - hi1.4xlarge
-      - hs1.8xlarge
-      - cr1.8xlarge
-      - cc2.8xlarge
-      - cg1.4xlarge
-    ConstraintDescription: Must be a valid EC2 instance type.
-################################
-# Create AWS resources
-################################
-Resources:
-  ####################################################################
-  # Create IAM resources for ECS task execution
-  ####################################################################
-  EC2ResourceAccessRole:
-    Type: AWS::IAM::Role
-    Properties:
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Principal:
-              Service: ec2.amazonaws.com
-            Action:
-              - 'sts:AssumeRole'
-      Path: /
-      Policies:
-        - PolicyName: AmazonEC2ContainerRegistryFullAccess 
-          PolicyDocument:
+    AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Seems like this file is not properly formatted. Can you please check?

Reply:
Seems like there is a tab space for complete file.

Reply:
Yes, changed it.